src/mount.js

```diff
@@ -47,6 +61,11 @@ class MountDatastore extends Adapter {
     }
   }
 
+  /**
+   * @param {Key} key
+   * @param {Uint8Array} value
+   * @param {Options} [options]
```

Comment:
```suggestion
   * @param {Options | undefined} [options]
```

Reply:
what do you mean @Gozala ?
@vasco-santos why do you prefer this syntax? its inferred by `[]`